### PR TITLE
doc: make the "logical partitioning" description more generic

### PR DESCRIPTION
**Pull request by gvancuts**
Make the description of the "Logial Partitioning" scenario more
generic than what is shown on the figure. This also helps as the
current examples of that scenario in the code base do not use
Safety or RTVM at the moment (as shown on the picture).

Tracked-On: #5903
Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

File: doc/introduction/index.rst [hunk 1]
```diff
@@ -341,9 +341,10 @@ non-real-time tasks.
 Logical Partition Scenario
 ==========================
 
-This scenario is a simplified VM configuration for VM logical
-partitioning: one is the Safety VM and the other is a Linux-based User
-VM.
+This scenario is a simplified configuration for VM logical
+partitioning: both User VMs are independant and isolated, they do not share
```
Comment (dbkinder) [inline doc/introduction/index.rst:345]: independent (with an 'e')

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/introduction/index.rst [hunk 2]
```diff
@@ -341,9 +341,10 @@ non-real-time tasks.
...
+resources and both are automatically launched at boot time by the hypervisor.
+The User VMs can be Real-Time VMs (RTVMs), Safety VMs or standard User VMs.
```
Comment (fitchbe) [inline doc/introduction/index.rst:347]: Serial comma after "resources", and one after "Safety VMs".

**Review by fitchbe — APPROVED** [review]

**Comment by gvancuts** [comment]
Addressed the comments and force-pushed the changes. PTAL.

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Comment by NanlinXie** [comment]
Currently it's impossible to have two safety VM, we may need fix the picture firstly and maybe we could add a clause at the end highlighting the constraint on the number of safety VMs / RTVMs that can coexist in one scenario.

**Comment by dbkinder** [comment]
@gvancuts Do you want to address @NanlinXie comment now, or in another PR?

**Comment by gvancuts** [comment]
> @gvancuts Do you want to address @NanlinXie comment now, or in another PR?

I'll address it in this PR.

@NanlinXie , is this a hard limitation that's here to stay?

**Comment by junjiemao1** [comment]
@gvancuts Though not a hard limitation in ACRN, having at most one safety VM is typical and highly recommended in safety-related system architecture and this is what we are exploring in safety uses. I would recommend to fix on this before we get a clear idea on the implication of having multiple safety VMs in a single scenario.

**Comment by gvancuts** [comment]
I have updated the picture in this latest version of the Pull Request. Can you all please take a look?

**Comment by fitchbe** [comment]
The new picture looks good to me.

**Comment by dbkinder** [comment]
OK to verify

**Pull request merged**